docs/plasma_calculator/index.rst

```diff
@@ -0,0 +1,24 @@
+.. _plasmapy-calculator:
+
+=================
+Plasma Calculator
+=================
+
+.. currentmodule:: plasmapy.utils.calculator
+
+
+Overview
+--------
+
+Plasma calculator is an interactive notebook installed alongside plasmapy that
+allows users to input a set of plasma properties and get immediate output for
+various functions at once.
+
+Using Plasma Calculator
+-----------------------
+
+To invoke the app use `plasma_calculator`, by default opens in a browser bootstrapping
+the notebook in a light theme and uses port 8866 by default.
+
+`plasma-calculator` takes optional arguments such as `dark`, `port`, `no-browser` pass flag
+`-h` or `--help` to get full list of supported arguments.
```

Comment:
```suggestion
``plasma-calculator`` takes optional arguments such as ``--dark``, ``--port``,
and ``--no-browser``.  Pass flag ``-h`` or ``--help`` to get the full list of 
supported arguments.
```
These flags need a `--` beforehand, I think?  I also added double back ticks so that the text inside gets interpreted as an in-line code snippet, since single back ticks in reST are used to link to a particular code object.